src/System.Management.Automation/FormatAndOutput/common/StringDecorated.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
```suggestion
#nullable enable
```

Reply:
Why not opt into nullable reference types when writing new code?

Reply:
Not using any nullable ref types in that file, is it a best practice to enable it by default?  I haven't really used it.

Reply:
It is probably the greatest addition to C# since generics in v2!

It documents if, for example, the strings you are using can be null or not.

with #nullable enable
`string? _text` means that it is OK for the field to be null
`string _text` means that it should never be null.

I've used it for a couple of months at work now, and I'm amazed by the clarity and quality it provides.
Something that should have been in the language, to begin with, and an engineering feat to accomplish after the fact. I'm really blown away by it.

Now, it's the first thing I enable when I get to a new codebase :) 

I actually think that we should enable it in the project, and add
`#nullable disable` to all files, and then start to remove the pragmas, instead of the other way around.

Below are two links worth reading.
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/nullable-reference-types
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/upgrade-to-nullable-references